librz/io/p/io_w32dbg.c

```diff
@@ -237,20 +167,9 @@ static int __close(RzIODesc *fd) {
 
 static char *__system(RzIO *io, RzIODesc *fd, const char *cmd) {
 	W32DbgWInst *wrap = fd->data;
-	//printf("w32dbg io command (%s)\n", cmd);
-	/* XXX ugly hack for testing purposes */
 	if (!strcmp(cmd, "")) {
 		// do nothing
 	} else if (!strncmp(cmd, "pid", 3)) {
-		if (cmd[3] == ' ') {
-			int pid = atoi(cmd + 3);
-			if (pid > 0 && pid != wrap->pi.dwThreadId && pid != wrap->pi.dwProcessId) {
-				wrap->pi.hThread = OpenThread(PROCESS_ALL_ACCESS, FALSE, pid);
-				if (!wrap->pi.hThread) {
-					eprintf("Cannot attach to %d\n", pid);
-				}
-			}
-		}
 		return rz_str_newf("%lu", wrap->pi.dwProcessId);
 	} else {
```

Comment:
is this command line coming from the rizin shell?

Reply:
its the `R!` command